### DISCUSSION TITLE
Slugify not in the strapi marketplace?

### DISCUSSION THREAD

**Discussion by christher-lenander** [ComfortablyCoding/strapi-plugin-slugify#31] [General]
I saw that Slugify was not in the marketplace anymore? is it because bug fixing?

**Comment by ComfortablyCoding** [comment]
No, it was unfortunately removed (temporarily) due to not respecting the model permissions per #29 . I have been attempting to find a way to fix it but have had no luck. The PR in question does not look to work atm either.

**Comment by ComfortablyCoding** [comment]
The plugin should be back on the market soon, #32 and #33 fixed the security issue reported.
